isis/tests/FunctionalTestsApolloremrx.cpp

```diff
@@ -0,0 +1,110 @@
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "apolloremrx.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+using ::testing::HasSubstr;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/apolloremrx.xml").expanded();
+
+TEST_F(ApolloCube, FunctionalTestApolloremrxDefault) {
+
+  testCube->label()->findObject("IsisCube").findGroup("Reseaus")["Status"] = "Refined";
+
+  QTemporaryDir prefix;
+  QString outCubeFileName = prefix.path()+"/outTEMP.cub";
+  QVector<QString> args = {"to=" + outCubeFileName,"action=null"};
+
+  UserInterface options(APP_XML, args);
+  try {
+    apolloremrx(testCube, options);
+  }
+  catch (IException &e) {
+    FAIL() << "Call failed, Unable to process cube: " << e.what() << std::endl;
+  }
+
+  Pvl newLab = *testCube->label();
```

Comment:
Should we be opening the output cube and testing its keywords instead of `testCube`